### PR TITLE
Default image

### PR DESCRIPTION
**Pull request by Steinbach2019**
the changes for Header/Photo Uploader set Text as variable

### CODE REVIEW

File: src/components/Form/PhotoUpload.vue [hunk 1]
```diff
@@ -15,16 +16,16 @@
...
+            <div class="q-uploader__title">{{$t(defaultText)}}</div>
             <div class="q-uploader__subtitle">
               {{ scope.uploadSizeLabel }} /
               {{ scope.uploadProgressLabel }}
             </div>
           </div>
           <q-btn v-if="scope.canAddFiles" type="a" icon="add_box" round dense flat>
             <q-uploader-add-trigger />
-            <q-tooltip>{{$t('Choose Photo')}}</q-tooltip>
+            <q-tooltip>{{$t('Your Logo')}}</q-tooltip>
```
Comment (cbleek) [inline src/components/Form/PhotoUpload.vue]: The ToolTip is shown on hover at the "+" Icon. 

By clicking the "+" Icon, the File-Selector Dialog of the Browser opens. So I would say, the text should contain "Choose" or "select"

File: src/components/Form/PhotoUpload.vue [hunk 2]
```diff
@@ -102,7 +103,7 @@ import VueCropper from 'vue-cropperjs'
 
 export default {
   name: 'PhotoUpload',
-  props: ['color', 'default-image', 'multiple'],
+  props: ['color', 'default-image', 'multiple', 'default-text'],
```
Comment (cbleek) [inline src/components/Form/PhotoUpload.vue]: maybe we should replace "default-text" by a parameter like "upload-type" which can take the values "Photo", "Logo", "Header Image"

In the translation files we yould have

```
"Select",
"Drag&Drop",
"Foto",
"Logo",
"Header Image",
```

Comment (Steinbach2019) [reply src/components/Form/PhotoUpload.vue]: i agree this comments => upload-type sounds more precise, i'll take the changes

Comment (Steinbach2019) [reply src/components/Form/PhotoUpload.vue]: the logic for upload-type is set up

**Pull request merged**